libs/components/list-builder-view-checklist/src/lib/modules/list-view-checklist/fixtures/list-view-checklist-toolbar.component.fixture.ts

```diff
@@ -32,8 +32,8 @@ export class ListViewChecklistToolbarTestComponent {
   }
 
   public changeVisibleItems(checked: boolean): void {
-    const checkbox = new SkyCheckboxChange();
-    checkbox.checked = checked;
-    this.checklist.changeVisibleItems(checkbox);
+    this.checklist.changeVisibleItems({
+      checked,
+    } as SkyCheckboxChange);
```

Comment:
Since the parameter for `changeVisibleItems()` is already typed `SkyCheckboxChange`, there's no need to cast the value here. TypeScript will infer the type from the parameter type and enforce type checking that way.